llvm/test/CodeGen/PowerPC/patchable-function-entry.ll

```diff
@@ -0,0 +1,49 @@
+; RUN: llc -mtriple=powerpc-unknown-linux-gnu %s -o - | FileCheck %s --check-prefixes=CHECK,PPC32
```

Comment:
`-mtriple=powerpc`. Prefer a generic ELF triple to a Linux specific one.

Suggest 2 RUN lines, `powerpc` and `powerpc64le`, to catch both 32-bit/64-bit and LE/BE.